docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -20,6 +29,8 @@ Testing
 ~~~~~~~
 * Decorator :py:func:`pvlib.conftest.fail_on_pvlib_version` can now be
   applied to functions that require args or kwargs. (:pull:`973`)
+* Test added for :py:meth:`ModelChain.ac_model` to confirm ValueError when
+  ac_model is an invalid string. (:pull:`886`)
```

Comment:
Nitpick: this renders as `ModelChain.ac_model()`, which might imply the check happens when the AC model is run but it actually happens when the `ac_model` string is stored by the ModelChain constructor.  Might be clearer to just link ```:py:class:`ModelChain` ``` instead.